### ISSUE TITLE
avn could not activate node

### ISSUE THREAD

**Issue by markahon** [wbyoung/avn#11]
I have an issue similar to #7. Any hints or ideas what might be causing this?

```
$ cat .node-version
v0.10.38

$ _avn explain -v .
avn could not activate node v0.10.38
error: no plugin passed predicate
  avn-nvm: nvm exited with status: 127
  avn-n: no version matching v0.10.38
```

**Comment by wbyoung** [comment]
What version of `n` are you using (`n --version`)?

**Comment by markahon** [comment]
Actually I'm not using `n`, just `nvm`. I just went and installed both `avn-n` and `avn-nvm` as the main readme suggested. I can't access nvm version atm, but it should be the latest as I installed it today.

**Comment by wbyoung** [comment]
@markahon the error that you're getting suggests that nvm is not installed properly (also consistent with not being able to do `nvm --version`). Is it sourced in your `.bash_profile` or other shell startup script?

**Comment by markahon** [comment]
I meant previously, that I couldn't access my laptop to check the version. It was sourced and worked otherwise. My nvm version was `0.23.3`, and I updated it to version `0.24` and tried again, but the problem still persisted.

Eventually I managed to make things work after some cleanup, but can't really tell what was the root of the problem. But what I did was:

1. Uninstalled `nvm`.
1. Uninstalled system `node` that I had installed with `homebrew`.
1. Installed `n` (version `1.3.0`).
1. Installed node versions `0.10` and `0.12` using `n`.
1. Installed `avn` and `avn-n`, and run `avn-setup`.

My best guess is that I hadn't run `avn setup` (even though I ran it several times) after messing up with different versions of `node` using nvm.
